packages/@uppy/url/src/UrlUI.js

```diff
@@ -36,7 +29,8 @@ class UrlUI extends Component {
         aria-label={this.props.i18n('enterUrlToImport')}
         placeholder={this.props.i18n('enterUrlToImport')}
         onkeyup={this.handleKeyPress}
-        ref={(input) => { this.input = input }} />
+        ref={(input) => { this.input = input }}
+        data-uppy-super-focusable />
```

Comment:
`ref={(input)` not needed here?

Reply:
It's not, Dashboard will see `data-uppy-super-focusable` attribute rendered, and automatically focus on it.

Reply:
Ok, right, but I guess it was left here by mistake then? Could you remove?

Reply:
@arturi, ah, it certainly was, I will! #TODO

Reply:
It wasn't actually, turns out `this.input` is used in that component.

Reply:
Oh, ok, sorry about that then!